requirements/edx/base.txt

```diff
@@ -47,7 +47,8 @@ meliae==0.4.0
 mongoengine==0.10.0
 MySQL-python==1.2.5
 networkx==1.7
-nose==1.3.7
+git+https://github.com/edx/nose.git@99c2aff0ff51bf228bfa5482e97e612c97a23245#egg=nose==1.3.7
```

Comment:
Maybe I missed this, but why does this need to be installed this way?  If this isn't actually v1.3.7, maybe is should just say `egg=nose` instead of `egg=nose==1.3.7`?